internal/cri/server/images/image_remove.go

```diff
@@ -34,17 +34,20 @@ import (
 // Remove the whole image no matter the it's image id or reference. This is the
 // semantic defined in CRI now.
 func (c *GRPCCRIImageService) RemoveImage(ctx context.Context, r *runtime.RemoveImageRequest) (*runtime.RemoveImageResponse, error) {
+	return c.CRIImageService.RemoveImage(ctx, r.GetImage())
+}
+
+func (c *CRIImageService) RemoveImage(ctx context.Context, imageSpec *runtime.ImageSpec) (*runtime.RemoveImageResponse, error) {
```

Comment:
IMO this function may just need to return a single `error` type rather than the additional `RemoveImageResponse` which is a CRI type. Similar pattern can be found for [ImagePull](https://github.com/containerd/containerd/blob/main/internal/cri/server/images/image_pull.go#L118) function too.